Xamarin.Forms.Platform.WPF/Renderers/ImageRenderer.cs

```diff
@@ -6,184 +6,185 @@
 using System.Windows;
 using System.Windows.Media;
 using System.Windows.Media.Imaging;
+using Standard;
 using Xamarin.Forms.Internals;
 using WImage = System.Windows.Controls.Image;
 
 namespace Xamarin.Forms.Platform.WPF
 {
-	public class ImageRenderer : ViewRenderer<Image, WImage>
```

Comment:
fix formatting

Reply:
I applied the formatting which changed the formatting a bit but it should fit to your general formatting now.